.github/workflows/pr.yml

```diff
@@ -19,10 +19,6 @@ jobs:
         run: npm install -g jest
       - name: Install NYC globally
         run: npm install -g nyc
-      - name: Run lint
```

Comment:
@lognathparamasivam why remove this ?

Reply:
Due to the dependency injection change, test cases were failing. I need to rework on all the testing files. I thought to take it up on last once I completed the functionality part.